src/schedule/zephyr_dp_schedule.c

```diff
@@ -273,7 +279,7 @@ static int scheduler_dp_task_free(void *data, struct task *task)
 	struct task_dp_pdata *pdata = task->priv_data;
 	int ret;
 
-	scheduler_dp_task_cancel(data, task);
+	scheduler_dp_task_stop(data, task);
```

Comment:
This is somewhat confusing with commit message as the patch both removes one call to scheduler_dp_task_cancel() and also claims it is not used. But ack, I now get you mean task_cancel() is not called via the ops.